it/controllers/employment/CheckYourBenefitsControllerISpec.scala

```diff
@@ -662,6 +662,7 @@ class CheckYourBenefitsControllerISpec extends IntegrationTest with ViewHelpers
   val companyCarHref: String = s"/income-through-software/return/employment-income/${defaultTaxYear-1}/benefits/company-car?employmentId=001"
 
   val companyVanBenefitsHref: String = s"/income-through-software/return/employment-income/${defaultTaxYear-1}/benefits/company-van?employmentId=001"
+  val mileageAmountBenefitsHref: String = s"/income-through-software/return/employment-income/${defaultTaxYear-1}/benefits/mileage-amount?employmentId=001"
```

Comment:
do you wanna put this inside the hrefs object instead? i know the ones around you arent so need moving at a later stage

Reply:
can tidy this up once we've done the car van fuel section I think